### ISSUE TITLE
Extension stop working since last update

### ISSUE THREAD

**Issue by NoksC** [tmizu23/bezierEditing#35]
Hi there,

Since the last update 1.3.6 of Bezier Editing, it's not working anymore, tried reinstall failed with following output :

```
ImportError: cannot import name 'QgsSettingsRegistryCore' 
Traceback (most recent call last):
  File "/usr/share/qgis/python/qgis/utils.py", line 335, in _startPlugin
    plugins[packageName] = package.classFactory(iface)
  File "/home/noks/.local/share/QGIS/QGIS3/profiles/default/python/plugins/BezierEditing/__init__.py", line 19, in classFactory
    from .bezierediting import BezierEditing
  File "/usr/share/qgis/python/qgis/utils.py", line 799, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/noks/.local/share/QGIS/QGIS3/profiles/default/python/plugins/BezierEditing/bezierediting.py", line 27, in 
    from .beziereditingtool import BezierEditingTool
  File "/usr/share/qgis/python/qgis/utils.py", line 799, in _import
    mod = _builtin_import(name, globals, locals, fromlist, level)
  File "/home/noks/.local/share/QGIS/QGIS3/profiles/default/python/plugins/BezierEditing/beziereditingtool.py", line 22, in 
    from qgis.core import QgsSettingsRegistryCore, QgsSettingsEntryBool, QgsWkbTypes, QgsProject, QgsVectorLayer, QgsGeometry, QgsPointXY, QgsFeature, QgsEditFormConfig, QgsFeatureRequest, QgsDistanceArea, QgsRectangle, QgsVectorLayerUtils, Qgis, QgsAction, QgsApplication, QgsMapLayer, QgsCoordinateTransform, QgsExpressionContextScope, QgsSettings, QgsMarkerSymbol, QgsTextAnnotation, QgsMessageLog
ImportError: cannot import name 'QgsSettingsRegistryCore'


Version de Python : 3.6.15 (default, Sep 23 2021, 15:41:43) [GCC] 
Version de QGIS : 3.18.3-Zürich Zürich, exported 

Chemin Python :
/usr/share/qgis/python
/home/noks/.local/share/QGIS/QGIS3/profiles/default/python
/home/noks/.local/share/QGIS/QGIS3/profiles/default/python/plugins
/usr/share/qgis/python/plugins
/usr/lib/python36.zip
/usr/lib64/python3.6
/usr/lib64/python3.6/lib-dynload
/usr/lib64/python3.6/site-packages
/usr/lib64/python3.6/site-packages/PIL
/usr/lib64/python3.6/_import_failed
/usr/lib/python3.6/site-packages
/home/noks/.local/share/QGIS/QGIS3/profiles/default/python
```

Thanks for your help, and please, excuse my bad english.

**Comment by tmizu23** [comment]
Thanks for the report.

BezierEditing version 1.3.6  requires QGIS 3.20 or later.
I did not change qgisMinimumVersion, I will fix it in the next release.

BezierEditing version 1.3.4 is required when using QGIS 3.18 or earlier.
To use BezierEditing version 1.3.4, download BezierEditing.zip from the following URL,
Select the zip file from Manage Plugins in QGIS and install it.

https://github.com/tmizu23/BezierEditing/releases/tag/v1.3.4

**Comment by NoksC** [comment]
Thanks a lot for your quick answer, problem solved with 1.3.4
